pkg/controller/cloud/node_controller.go

```diff
@@ -45,13 +45,14 @@ import (
 	nodeutil "k8s.io/kubernetes/pkg/util/node"
 )
 
-var UpdateNodeSpecBackoff = wait.Backoff{
+var updateNodeSpecBackoff = wait.Backoff{
 	Steps:    20,
 	Duration: 50 * time.Millisecond,
 	Jitter:   1.0,
 }
 
-type CloudNodeController struct {
+// NodeController is an object for controlling and monitoring cloud nodes.
+type NodeController struct {
```

Comment:
Just for the sake of consistency, should we rename \*Controller -> controller and New\*Controller -> NewController like we are doing in https://github.com/kubernetes/kubernetes/pull/68882?
@deads2k @sjgllghr @awly Does that make sense to you?

Reply:
There are 2 controllers in this package: http://go/gh/kubernetes/kubernetes/blob/70de955bf07e29dcce5595ad4ebe3628bdd201f5/pkg/controller/cloud/pvlcontroller.go#L55
We need to disambiguate.